doc/api/worker_threads.md

```diff
@@ -518,6 +521,8 @@ changes:
   If `options.eval` is `true`, this is a string containing JavaScript code
   rather than a path.
 * `options` {Object}
+  * `args` {string[]} List of string arguments which would be appended to
```

Comment:
should we have a one liner about the usage of / consideration between `workerData` versus `args`?